OSDOCS-8707: clarification for external gateway (#71305)
openshift/openshift-docs: modules/nw-secondary-ext-gw-about.adoc

[id="nw-secondary-ext-gw-about_{context}"]
= How {product-title} determines the external gateway IP address

> You configure a secondary external gateway with the `AdminPolicyBasedExternalRoute` custom resource from the the `k8s.ovn.org` API group. The custom resource (CR) supports static and dynamic approaches to specifying an external gateway's IP address.

Reviewer: ```suggestion
You configure a secondary external gateway with the `AdminPolicyBasedExternalRoute` custom resource (CR) from the the `k8s.ovn.org` API group. The CR supports static and dynamic approaches to specifying an external gateway's IP address.
```

Suggest defining CR on the first use

> Each namespace that a `AdminPolicyBasedExternalRoute` CR targets cannot be selected by any other `AdminPolicyBasedExternalRoute` CR. A namespace cannot have concurrent secondary external gateways. Because changes to policies are isolated in the controller, when a policy fails to apply, any changes to other policies will not trigger a retry to other policies. Only when a updates to the policy itself or related objects to the policy such as target namespaces, pod gateways, or namespaces hosting them from dynamic hops are made does the policy get re-evaluated applying any differences that might have occurred by the change.

Reviewer: ```suggestion
Each namespace that a `AdminPolicyBasedExternalRoute` CR targets cannot be selected by any other `AdminPolicyBasedExternalRoute` CR. A namespace cannot have concurrent secondary external gateways.

Changes to policies are isolated in the controller. If a policy fails to apply, changes to other policies do not trigger a retry of other policies. Policies are only re-evaluated, applying any differences that might have occurred by the change, when updates to the policy itself or related objects to the policy such as target namespaces, pod gateways, or namespaces hosting them from dynamic hops are made.
```

some suggestions for readability and present tense
